model/flow/execution_result.go

```diff
@@ -17,20 +18,45 @@ type ExecutionResult struct {
 	ExecutionDataID  Identifier // hash commitment to flow.BlockExecutionDataRoot
```

Comment:
Missing the `//structwrite:immutable - mutations allowed only within the constructor` annotation for ExecutionResult